src/chaos.js

```diff
@@ -52,4 +91,64 @@ export class KubernetesClient {
         this.kubernetes.namespaces.delete(randomNamespace.name)
         return randomNamespace.name
     }
-}
\ No newline at end of file
+}
+
+export class StressNodesAttack {
+    constructor(client) {
+        this.kubernetes = client
+        this.name = 'stress-nodes'
+        this.namespace = 'default'
+        this.cpu = '0'
+        this.io = '0'
+        this.vm = '0'
+        this.vmBytes = '256MB'
+        this.timeout = '10'
+    }
+
+    inNamespace(namespace) {
+        this.namespace = namespace
+        return this
+    }
+    inNodes(nodes) {
+        this.nodes = nodes
+        return this
+    }
+    withName(name) {
+        this.name = name
+        return this
+    }
+    withCPU(cpu) {
+        this.cpu = cpu
+        return this
+    }
+    withIO(io) {
+        this.io = io
+        return this
+    }
+    withVM(vm) {
+        this.vm = vm
+        return this
+    }
+    withVMBytes(vmBytes) {
+        this.vmBytes = vmBytes
+        return this
+    }
+    withTimeout(timeout) {
+        this.timeout = timeout
+        return this
+    }
+    start() {
+        this.kubernetes.runJobOnNodes(
+            this.namespace,
+            this.name,
+            this.nodes,
+            'progrium/stress',
+            ['stress', '--cpu', this.cpu, '--io', this.io, '--vm', this.vm, '--vm-bytes', this.vmBytes, '--timeout', this.timeout]
```

Comment:
I think it would be necessary to set the resource limits for the pods generated by the jobs to prevent them to be evicted or throttled. Otherwise, the load they would impose in the node would be less than expected.